openedx/features/idea/forms.py

```diff
@@ -8,37 +8,39 @@
 from lms.djangoapps.onboarding.models import Organization
 from openedx.features.philu_utils.utils import validate_file_size
 
-from .constants import COUNTRY_MAX_LENGTH, IDEA_FILE_MAX_SIZE, IDEA_IMAGE_MAX_SIZE, ORGANIZATION_NAME_MAX_LENGTH
+from .constants import IDEA_FILE_MAX_SIZE, IDEA_IMAGE_MAX_SIZE, ORGANIZATION_NAME_MAX_LENGTH
 from .helpers import validate_image_dimensions
 from .models import Idea
 
 
 class IdeaCreationForm(ModelForm):
     """
-    A model form to create new Idea. Separate fields for organization_name and country_name are used
-    to make model field organization and country work with custom widget on frontend.
+    A model form to create new Idea. Separate fields for organization_name is used
+    to make model field organization work with custom widget on frontend.
     """
 
     prefix = 'idea'
     label_suffix = ''
 
     organization_name = CharField(
         max_length=ORGANIZATION_NAME_MAX_LENGTH,
-        label=_('Organization Name'),
-        required=True,
-        widget=TextInput(),
-    )
-
-    country_name = CharField(
```

Comment:
Since country label is changed we need to update test data at `edx-platform/openedx/features/idea/tests/data/test_data_idea.json`

Reply:
Done